seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/config/HbaseConfig.java

```diff
@@ -97,6 +97,9 @@ public class HbaseConfig {
                     .noDefaultValue()
                     .withDescription("Hbase extra config");
 
+    public static final Option<Long> HBASE_TTL_CONFIG =
+            Options.key("ttl").longType().defaultValue(-1L).withDescription("Hbase ttl config");
```

Comment:
```suggestion
            Options.key("ttl").longType().defaultValue(-1L).withDescription("The expiration time configuration for writing hbase data. The default value is -1, indicating no expiration time.");
```

Reply:
> 

Thank you for your guidance. The modifications have been completed

Reply:
https://github.com/apache/seatunnel/pull/7116#pullrequestreview-2161342401

Reply:
> [#7116 (review)](https://github.com/apache/seatunnel/pull/7116#pullrequestreview-2161342401)

Add Complete